### PR TITLE
New blog post on ProcDump in Fedora

### PR DESCRIPTION
**Pull request by mgrabovsky**

### CODE REVIEW

File: _posts/2019-10-24-procdump-fedora.markdown [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+And then there's the classic ProcDump whose [Linux port][github] is now available in
+Fedora repositories for the benefit of everybody.
+
+If you've ever tried to debug a sneaky performance bug, you know it's not so easy to
+do by hand. [ProcDump][msft] was originally created to watch for spikes in CPU usage
+of Windows applications and to create core dumps when these spikes hit. It was first
+released in 2009 and is since being developed as part of the Windows Sysinternals
+family.
+
+A Linux port called [ProcDump for Linux][github] has been released some two years ago
```
Comment (xsuchy) [inline _posts/2019-10-24-procdump-fedora.markdown]: had been

**Comment by mkutlak** [comment]
LGTM

**Comment by mgrabovsky** [comment]
I will be reworking it slightly, so don't merge yet please.

**Comment by ernestask** [comment]
LGTM

**Pull request merged**